bindings/rust/s2n-tls/src/client_hello.rs

```diff
@@ -168,7 +168,7 @@ impl ClientHello {
         Ok(session_id)
     }
 
-    fn server_name(&self) -> Result<Vec<u8>, Error> {
+    pub fn server_name(&self) -> Result<Vec<u8>, Error> {
```

Comment:
Should raw_message below be pub too? Can you double-check the visibility of all these methods?

Reply:
Yeah probably. No idea why raw_message wasn't public in the first place. 🤔